### PR TITLE
APP-3013: Added a CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by symphony-elias**

### CODE REVIEW

File: docs/pul_request_template.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+### Ticket
```
Comment (symphony-youri) [inline docs/pul_request_template.md:1]: the file name is missing an l (pul-> pull)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# How to contribute
+
+If you found a bug or issue, please open a GitHub issue [here](https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/issues).
+If you open a PR to fix any issue, please reference the ticket in the PR title.
+A [Symphony SDK team](https://github.com/orgs/SymphonyPlatformSolutions/teams/symphony-sdk/members)
```
Comment (symphony-youri) [inline CONTRIBUTING.md:5]: team member?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+will have to approve before it is merged and eventually released.
+
+If you want to request an enhancement or feature, please reach out to Symphony support.
```
Comment (symphony-youri) [inline CONTRIBUTING.md:8]: how do we reach the support? (or should we let people use the issues for that too? this way the conversation is public and people can look at already closed issues)

File: docs/pul_request_template.md (hunk 1 above)
Comment (symphony-youri) [reply docs/pul_request_template.md:1]: I would perhaps use the path .github/pull_request_template.md to make it hidden (and we already have this .github folder)

**Review by symphony-youri — APPROVED** [review]

**Review by symphony-youri — APPROVED** [review]

**Review by symphony-thibault — APPROVED** [review]

**Pull request merged**